specification/storage/resource-manager/Microsoft.Storage/preview/2018-03-01-preview/examples/StorageAccountSetManagementPolicy.json

```diff
@@ -0,0 +1,136 @@
+{
+  "parameters": {
+    "subscriptionId": "45b60d85-fd72-427a-a708-f994d26e593e",
+    "resourceGroupName": "res7231",
+    "accountName": "sto288",
+    "managementPolicyName": "default",
+    "api-version": "2018-03-01-preview",
+    "monitor": "true",
+    "properties": {
+      "properties": {
+        "policy": {
+          "version": 0.5,
+          "rules": [
+            {
+              "name": "ruleFoo",
+              "type": "lifecycle",
+              "definition": {
+                "filters": {
+                  "blobTypes": [
+                    "blockBlob"
+                  ],
+                  "nameMatch": [
+                    "foo"
+                  ]
+                },
+                "actions": {
+                  "baseBlob": {
+                    "tierToCool": {
+                      "daysAfterLastModifiedGreaterThan": 30
+                    },
+                    "tierToArchive": {
+                      "daysAfterLastModifiedGreaterThan": 90
+                    },
+                    "delete": {
+                      "daysAfterLastModifiedGreaterThan": 2555
+                    }
+                  },
+                  "snapshot": {
+                    "delete": {
+                      "daysAfterCreationGreaterThan": 90
+                    }
+                  }
+                }
+              }
+            },
+            {
+              "name": "expirationRule",
+              "type": "Lifecycle",
+              "definition": {
+                "filters": {
+                  "blobTypes": [
+                    "blockBlob"
+                  ]
+                },
+                "actions": {
+                  "baseBlob": {
+                    "delete": {
+                      "daysAfterLastModifiedGreaterThan": 365
+                    }
+                  }
+                }
+              }
+            }
+          ]
+        }
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/45b60d85-fd72-427a-a708-f994d26e593e/resourceGroups/res7231/providers/Microsoft.Storage/storageAccounts/sto288/managementPolicies/default",
+        "name": "DefaultManagementPolicy",
+        "type": "Microsoft.Storage/storageAccounts/managementPolicies",
```

Comment:
Why does your server returns different name for the Resource object than it was provided in request and from what appears in the URL? Does it always add `ManagementPolicy` suffix?

Reply:
Actually， We don't provide "name" property in request. 
The "managementPolicyName": "default" is just input with URI，and it's a hardcode value. We use it as input just for generate schema with it. (if the "default" is not defined as a input parameter in URI, the schema won't be generated with the Put API)

And for other "name" in the request, they are just part of the policy, and are differenct with the responds "Name". 

For the Name, id, Type in the responds, that's stardard properties of a SRP object.